contrib/duckdb-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+option(ENABLE_DUCKDB "Enable duckdb" ${ENABLE_LIBRARIES})
+
+set (DUCKDB_LIBRARY_DIR "${ClickHouse_SOURCE_DIR}/contrib/duckdb-amalgamation")
+
+set (SRCS
```

Comment:
I don't the compilation of sources related to `fsst`, maybe they need to be included manually?

Reply:
Don't we include fsst sources in the line 18 with `"${DUCKDB_LIBRARY_DIR}/duckdb-fsst.cpp"`?

Reply:
Ok, I overlooked it...